app/models/candidate_interface/sign_up_form.rb

```diff
@@ -0,0 +1,27 @@
+module CandidateInterface
+  class SignUpForm
+    include ActiveModel::Model
+
+    attr_accessor :email_address, :accept_ts_and_cs
+    validates :email_address, :accept_ts_and_cs, presence: true
+    validates :email_address, length: { maximum: 250 },
+                              format: { with: Candidate::NOTIFY_EMAIL_REGEXP }
+
+
+    def save_base(candidate)
```

Comment:
Reckon this can just be called `#save`. We've used `#save_base` when there's a need for multiple `#save` methods.